src/Sulu/Bundle/CoreBundle/DependencyInjection/SuluCoreExtension.php

```diff
@@ -33,39 +33,6 @@ class SuluCoreExtension extends Extension implements PrependExtensionInterface
      */
     public function prepend(ContainerBuilder $container)
     {
-        // process the configuration of SuluCoreExtension
-        $configs = $container->getExtensionConfig($this->getAlias());
-        $parameterBag = $container->getParameterBag();
-        $configs = $parameterBag->resolveValue($configs);
-        $config = $this->processConfiguration(new Configuration(), $configs);
-
-        if (isset($config['phpcr'])) {
-            $phpcrConfig = $config['phpcr'];
-
-            // TODO: Workaround for issue: https://github.com/doctrine/DoctrinePHPCRBundle/issues/178
-            if (!isset($phpcrConfig['backend']['check_login_on_server'])) {
-                $phpcrConfig['backend']['check_login_on_server'] = false;
-            }
-
-            foreach (array_keys($container->getExtensions()) as $name) {
-                $prependConfig = [];
-                switch ($name) {
-                    case 'doctrine_phpcr':
-                        $prependConfig = [
-                            'session' => $phpcrConfig,
-                            'odm' => [],
-                        ];
-                        break;
-                    case 'cmf_core':
-                        break;
-                }
-
-                if ($prependConfig) {
-                    $container->prependExtensionConfig($name, $prependConfig);
-                }
-            }
-        }
```

Comment:
I guess this has been moved to another bundle? That's not a real BC break, at least not if all bundles are registered, right?